docs/content/en/docs/getting-started/production-environment/_index.md

```diff
@@ -3,69 +3,94 @@ title: "Create production cluster"
 
-{{% alert title="Important" color="warning" %}}
-
-Creating an EKS Anywhere management cluster is the recommended model.
+Creating an EKS Anywhere management cluster is the recommended model for vSphere deployments.
 Separating management features into a separate, persistent management cluster
 provides a cleaner model for managing the lifecycle of workload clusters (to create, upgrade, and delete clusters), while workload clusters run user applications.
 This approach also reduces provider permissions for workload clusters.
 
 {{% /alert %}}
 
-## Prerequisite Checklist
+## Prerequisite checklist
 
 EKS Anywhere needs to be run on an administrative machine that has certain [machine
 requirements]({{< relref "../install" >}}).
-An EKS Anywhere deployment will also require the availability of certain
-[resources from your VMware vSphere deployment]({{< relref "/docs/reference/vsphere/vsphere-prereq/_index.md" >}}).
+An EKS Anywhere deployment will also require the availability of certain resources that vary depending on your provider:
+
+* [Bare Metal requirements]({{< relref "/docs/reference/baremetal/bare-prereq.md" >}})
+* [VMware vSphere requirements]({{< relref "/docs/reference/vsphere/vsphere-prereq.md" >}})
+
+## Preparation checklist
+
+With prerequisites in place, addional preparation is required, depending on your provider:
 
+* [Bare Metal preparation]({{< relref "/docs/reference/baremetal/bare-preparation.md" >}})
+* [VMware vSphere preparation]({{< relref "/docs/reference/vsphere/vsphere-preparation.md" >}})
+
+Also, see the [Ports and protocols]({{< relref "/docs/reference/ports.md" >}}) page for information on ports that need to be accessible from control plane, worker, and Admin machines.
 ## Steps
 
 The following steps are divided into two sections:
 
-* Create an initial cluster (used as a management or self-managed cluster)
-* Create zero or more workload clusters from the management cluster
+* Create an initial cluster 
+* Create zero or more workload clusters from the management cluster (vSphere only)
 
 ### Create an initial cluster
 
-Follow these steps to create an EKS Anywhere cluster that can be used either as a management cluster or as a self-managed cluster (for running workloads itself).
+Follow these steps to create an initial EKS Anywhere cluster.
 
 <!-- this content needs to be indented so the numbers are automatically incremented -->
-1. Generate an initial cluster config (named `mgmt` for this example):
+1. Set environment variables for your provider. For vSphere, these include the cluster name, and your vSphere user name and password (use single quotes around the values):
+   
+   ```bash
+   export CLUSTER_NAME=mgmt
+   export EKSA_VSPHERE_USERNAME='billy'
+   export EKSA_VSPHERE_PASSWORD='t0p$ecret'
+   ```
+   For Bare Metal, set the cluster name, IP address of the Tinkerbell stack, and the path to the hardware inventory (for example, `hardware.csv`):
+   ```bash
+   export PROVIDER=tinkerbell
```

Comment:
where does this get used?

Reply:
Removed when I split apart the getting started into two: vSphere and Bare Metal.